mreg/api/v1/tests/test_host_permissions.py

```diff
@@ -293,3 +294,15 @@ def test_super_only_delete_wildcard(self):
         path = '/api/v1/hosts/*.example.org'
         self.assert_delete_and_403(path)
         self.assert_delete(path, client=self.super_client)
+
+    def test_special_group_members_create_wildcard(self):
+        """Members in DNS_WILDCARD_GROUP can create entries with a wildcard, but only below subdomains"""
+        self.user = self.admin_user
```

Comment:
Can instead call:
`self.client = self.get_token_client(superuser=False, adminuser=False)` 
to get an client without extra access. Or is it intentional that you need to be both an admin _and_ `DNS_WILDCARD_GROUP`-member?

Reply:
Yes. If the user isn't admin, it needs a NetgroupRegexPermission to be able to create a host. So the test function would have to add a permission for the user.